src/isEmpty.js

```diff
@@ -16,6 +16,8 @@ var _curry1 = require('./internal/_curry1');
  *      R.isEmpty([]); //=> true
  *      R.isEmpty(''); //=> true
  *      R.isEmpty(null); //=> false
+ *      R.isEmpty({}); //=> false
```

Comment:
should we annotate this possibly surprising example to point out, e.g.:

`R.isEmpty(null); //=> false; {} is not a list.`